packages/in_app_purchase/in_app_purchase_android/example/lib/main.dart

```diff
@@ -22,7 +22,9 @@ void main() {
   runApp(_MyApp());
 }
 
-const bool _kAutoConsume = true;
+// Auto-consume must be true on iOS.
```

Comment:
This is the Android implementation package's example, which only supports Android, so this can just be removed.